po/CMakeLists.txt

```diff
@@ -0,0 +1,42 @@
+if (GETTEXT_FOUND)
+
+include(GNUInstallDirs)
+if(NOT CMAKE_INSTALL_LOCALEDIR)
+  MESSAGE(STATUS "CMAKE_INSTALL_LOCALEDIR not found, i18n files will not be installed")
+else()
+
+set(potfile "${CMAKE_CURRENT_BINARY_DIR}/${PROJECT_NAME}.pot")
+file(GLOB potfiles_in
+  RELATIVE "${CMAKE_CURRENT_SOURCE_DIR}/../src/"
+  "${CMAKE_CURRENT_SOURCE_DIR}/../src/*.h"
+  "${CMAKE_CURRENT_SOURCE_DIR}/../src/*.cc")
+
+add_custom_command(OUTPUT ${potfile}
+  COMMAND xgettext
+          --default-domain=${PROJECT_NAME}
+          --package-name=${PROJECT_NAME}
+          --package-version=${PROJECT_VERSION}
+          --copyright-holder="John Wiegley"
+          --msgid-bugs-address="https://bugs.ledger-cli.org"
+          --keyword='_'
+          --keyword='_f'
+          --keyword='_n:1,2'
+          --keyword='_fn:1,2'
```

Comment:
The keywords refer to the special custom macros for gettext functions defined in [`system.hh.in`:200ff](https://github.com/ledger/ledger/blob/5bd653948fa4d1e66e9b9b4d4ee1eb267c44385e/src/system.hh.in#L200:L204)